win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5256,14 +5785,19 @@ TextIntRealizeFont(HFONT FontHandle, PTEXTOBJ pTextObj)
         PSHARED_FACE SharedFace = FontGdi->SharedFace;
 
         TextObj->TextFace[0] = UNICODE_NULL;
+        IntLockFreeType();
         if (MatchFontNames(SharedFace, SubstitutedLogFont.lfFaceName))
         {
+            IntUnLockFreeType();
             RtlStringCchCopyW(TextObj->TextFace, _countof(TextObj->TextFace), pLogFont->lfFaceName);
         }
         else
         {
+            IntUnLockFreeType();
             RtlInitUnicodeString(&Name, NULL);
+            IntLockFreeType();
```

Comment:
Why is this unlock+lock done here? This looks pretty unefficient. Just keep the lock held.

Reply:
```suggestion
            RtlInitUnicodeString(&Name, NULL);
```